src/Score/computeRoundState_test/sword-challenge.js

```diff
@@ -7,11 +7,11 @@ import {
   SYMBOL_COIN,
 } from "/src/symbols/symbol-types.js"
 import { CARD_SWORD_CHALLENGE } from "src/Cards/card-types.js"
-import { computeScore } from "../ScoreHelpers.js"
+import { computeRoundState } from "../ScoreHelpers.js"
 
 // 8 coin
 {
-  const actual = computeScore({
+  const actual = computeRoundState({
```

Comment:
we should add a test ensuring 3 skulls or more means we get a negative score and not 0.

Reply:
Fixed in the last commit :+1: